lawlibrary/static/stylesheets/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$primary: #5a9d1c;
+
+$font-family-sans-serif: Lato, Roboto, "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+$font-size-base: 0.875rem;
```

Comment:
If these are the same as what's in peachjam's `_variables`, then they don't need to be re-stated here.